portal/component-tests/_macros/facility-stage.component-test.js

```diff
@@ -0,0 +1,40 @@
+const { FACILITY_STAGE } = require('@ukef/dtfs2-common');
+const componentRender = require('../componentRenderer');
+
+const component = '_macros/facility-stage.njk';
+const render = componentRender(component);
+const facilities = require('../fixtures/deal-fully-completed');
+
+describe('_macros/facility-stage.njk', () => {
+  it(`should render facility stage as ${FACILITY_STAGE.ISSUED}, where facility has been issued.`, () => {
+    // Act
+    const wrapper = render(facilities.loanTransactions.items[0]);
```

Comment:
I think I'd prefer this to be more explicit, something like

```suggestion
    const wrapper = render({ ...facilities.loanTransactions.items[0], facilityStage: FACILITY_STAGE.ISSUED });
```

Reply:
or does it need `hasBeenIssued: true` - it's not clear here whats the significant part

Reply:
This does not represent the true state, in the code the whole faciliy object is supplied. This is done explicity to keep it future proof.

Reply:
I think the entire true state can be passed in but its hard from looking at the tests to determine which is the significant value - is it `hasBeenIssued` or is it `facilityStage`. If this is clearly shown in the mock data it will be easier to maintain this test